src/services/helpers.ts

```diff
@@ -50,3 +50,15 @@ export const childrenLabel = (discipline: DisciplineType): string => {
 
 export const childrenDescription = (discipline: DisciplineType): string =>
   `${discipline.numberOfChildren} ${childrenLabel(discipline)}`
+
+/**
+ * Shuffles an array using the Durstenfeld algorithm as found here: https://stackoverflow.com/a/12646864
```

Comment:
@maxammann I'm not sure about JSX conventions, is this alright or do you have some suggestions?